CoreFoundation/PlugIn.subproj/CFBundle_Internal.h

```diff
@@ -30,20 +30,42 @@ CF_EXTERN_C_BEGIN
 #define PLATFORM_PATH_STYLE kCFURLPOSIXPathStyle
 #endif
 
+// Freestanding bundles are supported on all platforms.
+#if TRUE
```

Comment:
This should not be enabled on Darwin.

Reply:
I think this could be useful for things like CLI tools that may require some resources without being in an app bundle. In any case I will disable in Darwin for now.

Reply:
CLI that require resources have their own mechanism on Darwin. Freestanding bundles are for FHS OSes and Windows only.